### ISSUE TITLE
Migrate up fails when trying to migrate WordPress on Postgres

### ISSUE THREAD

**Issue by divyenduz** [prisma/migrate#500]
Report https://prisma-errors.netlify.app/report/8419 (latest report, clearer message)
Report https://prisma-errors.netlify.app/report/8419 (older report, less clear message)

## Bug description

Trying to migrate schema introspected from wordpress to a Postgres database fails

## How to reproduce

1. Get schema.prisma for Wordpress here: https://github.com/prisma/introspection-engine-output/blob/master/introspection-analysis/output/mysql_public/wordpress.log (internal)
2. Remove the `generator` block, replace the `datasource` block to be a Postgres database. 
2. `npx -p @prisma/cli@dev prisma migrate save --name init --experimental`
3. `npx -p @prisma/cli@dev prisma migrate up --experimental `

## Expected behavior

It should work or fail at a validation step

## Prisma information

```
divyendusingh [p2-migrate-wp]$ npx -p @prisma/cli@dev prisma --version                                                                                            130 ↵
@prisma/cli          : 2.2.0-dev.48
Current platform     : darwin
Query Engine         : query-engine ed51c9476579e3aa52ec43079fb2a351c70f5bb7 (at /Users/divyendusingh/.npm/_npx/760/lib/node_modules/@prisma/cli/query-engine-darwin)
Migration Engine     : migration-engine-cli ed51c9476579e3aa52ec43079fb2a351c70f5bb7 (at /Users/divyendusingh/.npm/_npx/760/lib/node_modules/@prisma/cli/migration-engine-darwin)
Introspection Engine : introspection-core ed51c9476579e3aa52ec43079fb2a351c70f5bb7 (at /Users/divyendusingh/.npm/_npx/760/lib/node_modules/@prisma/cli/introspection-engine-darwin)
Format Binary        : prisma-fmt ed51c9476579e3aa52ec43079fb2a351c70f5bb7 (at /Users/divyendusingh/.npm/_npx/760/lib/node_modules/@prisma/cli/prisma-fmt-darwin)

```

**Comment by tomhoule** [comment]
I just investigated this and there is a single issue preventing the migration from executing: the schema has several indexes called "meta_key" on different tables. This is introspected from MySQL and on MySQL it works, but on SQLite and Postgres, index names have to be globally unique. @mavilein  do you think this could/should be a schema validation? Otherwise we could validate that in the migration engine.

**Comment by mavilein** [comment]
Yeah this should be a schema validation imo. Probably a great task for someone to learn about the schema parser.

**Comment by tomhoule** [comment]
Recap comment: this is now a schema validation issue. We should validate this during datamodel validation to return fast and good errors.
